cpp/roaring.hh

```diff
@@ -112,6 +113,16 @@ public:
         return ans;
     }
 
+    /**
+     * Construct a bitmap from a list of uint32_t values.
+     * E.g., bitmapOf({1,2,3}).
```

Comment:
Typo (new spelling is bitmapOfList) and also spacing if you want

```
 * E.g., bitmapOfList({1, 2, 3}).
```

Reply:
Thanks. Fixed.